packages/ui-markdown-editor/src/utilities/matchCases.js

```diff
@@ -30,6 +34,87 @@ export const matchCases = (editor, currentLine) => {
 		return;
 	}
 
-	matchHeadings(editor, currentLine);
-	matchPageBreak(editor, currentLine);
-}
+	/**
+	 * 
+	 * @param {string} textToInsert The text that we want to format
+	 * @param {Object} textFormats This is the format style of the text (bold, italic, code) we want to apply or remove as the key in an object
+	 * and a boolean as the value. i.e. `{ bold: true }`
+	 * @param {Integer} markdownCharacters The number of markdown characters that the user has to type to trigger WYSIWYG
+	 */
+	const insertFormattedInlineText = (textToInsert, textFormats, markdownCharacters) => {
```

Comment:
Could `insertFormattedInlineText` be defined outside of `matchCases`?

Reply:
Should I put it outside the `matchCases` function or in another file?

Reply:
I think outside the function, probably keep it in this file for now.